splunk-otel-android/src/main/java/com/splunk/rum/SplunkRumBuilder.java

```diff
@@ -0,0 +1,323 @@
+/*
+ * Copyright Splunk Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.splunk.rum;
+
+import static com.splunk.rum.DeviceSpanStorageLimiter.DEFAULT_MAX_STORAGE_USE_MB;
+
+import android.app.Application;
+import android.util.Log;
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.sdk.trace.export.SpanExporter;
+import io.opentelemetry.semconv.resource.attributes.ResourceAttributes;
+import java.time.Duration;
+import java.util.concurrent.atomic.AtomicReference;
+import java.util.function.Consumer;
+
+/** A builder of {@link SplunkRum}. */
+public final class SplunkRumBuilder {
```

Comment:
> What if we delegated to the Config for now and did this work in the subsequent version that removes it?
> 
> It's pure duplication, right? So pure delegation should be safe/ok? I just hate to open us up to divergence....but it's probably fine.

Yeah, it would work perfectly fine. Should I do it?

Reply:
Your call. I don't want to create more work, and the deprecation will be here soon enough. 🤷🏻

Reply:
I'm kinda lazy, so I'm inclined to leave it as it is 😅